master-lps-63148

anthony-chu opened the pull request:
https://issues.liferay.com/browse/LPS-63148

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/914/">test-portal-acceptance-pullrequest(master)</a></p>

anthony-chu commented:
cc @kenjiheigel I've updated the UIInfrastructureUsecase#Smoke test to check for the 'Sharing' since the Permissions tab in the nav-bar has been extracted to its own menu item and iframe.

cc @sharonchoi I've updated tests in PGBlogs.testcase to navigate to Permission configuration using the menu item instead of opening the configuration then navigating through the navbar.

cc @ealonso This is a new pull request for #524.

Please let me know if you guys have any questions. Thanks!

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 45 minutes 25 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/da8bdc210dde073bba8623d50176056987c03333">da8bdc210dde073bba8623d50176056987c03333</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/914/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27193/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27195/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16859/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27564/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27198/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19121/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/11</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15238/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/12</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27638/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></li><li><strike><strong><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19122/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></strong></strike></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22797/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27565/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/16</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23808/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/17</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16916/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/18</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19123/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/19</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23807/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/2</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15281/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/20</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19124/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/21</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27567/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/22</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19125/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/23</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15587/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/3</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19631/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/4</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15280/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/5</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19120/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/6</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23738/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16860/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16861/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/9</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23809/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19126/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27568/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk7</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/24029/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/22799/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk7</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27644/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk7</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23797/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13023/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk7</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15347/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16920/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk7</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27201/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13024/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk7</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27202/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk7</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/1036/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-11.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/914/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/914/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>39 Jobs Passed.<br />2 Jobs Failed.</p><pre>Completed with the status of FAILURE.</pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19122/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></h5><h6>Job Results:</h6><p>18 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/19122//testReport/com.liferay.poshi.runner/PoshiRunner/test_PGDocumentsandmedia_AddDocument_">PoshiRunner.test[PGDocumentsandmedia#AddDocument]</a> - <a href="https://test-1-11.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/914/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/functional-tomcat8-mysql56-jdk7/14">Poshi Report</a> - <a href="https://test-1-11.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/914/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/functional-tomcat8-mysql56-jdk7/14">Poshi Summary</a> - <a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/19122//console">Console Output</a></li></ol></li></ol>

evannagayama commented:
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/36041
